### ISSUE TITLE
Cannot install dependencies

### ISSUE THREAD

**Issue by Citrullin** [tum-esi/wade#91]
When executing `npm install` I get the following error.

```bash
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated @hapi/joi@15.1.1: joi is leaving the @hapi organization and moving back to 'joi' (https://github.com/sideway/joi/issues/2411)
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated webdriverio@4.14.4: outdated version, please use @next
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated @hapi/bourne@1.3.2: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/topo@3.1.6: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/hoek@8.5.1: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/address@2.1.4: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> node-aead-crypto@2.2.0 preinstall /home/citrullin/git/wade/node_modules/node-aead-crypto
> node lib/preinstall.js

node-aead-crypto not needed on this system, since all ciphers are natively supported
npm WARN notsup Unsupported engine for node-aead-crypto@2.2.0: wanted: {"node":">4 <10"} (current: {"node":"12.18.3","npm":"6.14.6"})
npm WARN notsup Not compatible with your version of node/npm: node-aead-crypto@2.2.0
npm WARN notsup Unsupported engine for watchpack-chokidar2@2.0.0: wanted: {"node":"<8.10.0"} (current: {"node":"12.18.3","npm":"6.14.6"})
npm WARN notsup Not compatible with your version of node/npm: watchpack-chokidar2@2.0.0
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/watchpack-chokidar2/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/fork-ts-checker-webpack-plugin/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/webpack-dev-server/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-aead-crypto@2.2.0 preinstall: `node lib/preinstall.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-aead-crypto@2.2.0 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/citrullin/.npm/_logs/2020-09-23T15_08_12_671Z-debug.log
```

```bash
> node --version
v12.18.3
```

```bash
> npm --version
6.14.6
```

```bash
> lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 19.10
Release:	19.10
Codename:	eoan
```

on commit `82caa3d4a9da5fa6469950acb0b6b8f86a4ad0c7`

**Comment by egekorkan** [comment]
Could you try to force install node-aead-crypto ? We have documented it here: https://github.com/tum-esi/wade#errors-and-known-problems and also at node-wot https://github.com/eclipse/thingweb.node-wot/issues/136

If this is indeed the problem, I will push this troubleshoot somewhere more prominent :)

**Comment by Citrullin** [comment]
> Could you try to force install node-aead-crypto ? We have documented it here: https://github.com/tum-esi/wade#errors-and-known-problems and also at node-wot [eclipse/thingweb.node-wot#136](https://github.com/eclipse/thingweb.node-wot/issues/136)
> 
> If this is indeed the problem, I will push this troubleshoot somewhere more prominent :)

Maybe at some point it makes sense to update all these packages. The warning are quite concerning. Especially when we talk about Electron. Looks like there is a native API for these kind of purposes in node these days.
"Note: This module is not necessary on NodeJS 10+ because you can now provide the authTagLength"

**Comment by egekorkan** [comment]
I have tried to understand what is going on. Some possible solutions:
- Do you have node-gyp installed?
- Can you try to install electron-builder? This should not be necessary though

Until I understand how to fix it, you can use the following AppImage
https://we.tl/t-Fq3Hf780Yk

Not the most professional way to share binary files probably :)

**Comment by Citrullin** [comment]
> I have tried to understand what is going on. Some possible solutions:
> 
>     * Do you have node-gyp installed?
> 
>     * Can you try to install electron-builder? This should not be necessary though
> 
> 
> Until I understand how to fix it, you can use the following AppImage
> https://we.tl/t-Fq3Hf780Yk
> 
> Not the most professional way to share binary files probably :)

Probably not ^^
I guess you are also available at the PlugFest. Couldn't join yesterday, but I will be in the call today. So maybe a good opportunity to figure it out.

**Comment by relu91** [comment]
I can't install too, it seems that my sistem cannot find `vue-cli-service`. 😭 :
```bash
npm i vue-cli-service --save-dev
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/vue-cli-service - Not found
npm ERR! 404
npm ERR! 404  'vue-cli-service@latest' is not in 
the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from 
a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found 
in:
npm ERR!     C:\Users\reluc\AppData\Roaming\npm-cache\_logs\2020-09-29T07_55_11_723Z-debug.log  
```

**Comment by egekorkan** [comment]
Yes I will be there :)

**Comment by egekorkan** [comment]
So this is fixed via #95 . @Citrullin you can try again
